### PR TITLE
feat: integrate fee collector module

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #1198 

```
brownie run scripts/issue/1198/fee_module_integration enable_module

brownie run scripts/issue/1198/fee_module_integration add_member
```

I think that given the 2m in gas limit for this upkeep id, it may revert when adding the member, since it will require more eth funds. since already at 1m it will require ~121 link

### CODE REVIEW

**Review by gosuto-inzasheru — COMMENTED** [review]
- deployed contract and reviewed source code diff: diff: https://www.diffchecker.com/UOaNJ24V
- `enable_module` runs fine
- getting this on `add_member`??
```
Running 'scripts/issue/1198/fee_module_integration.py::add_member'...
Transaction sent: 0x7d57610ea5ea1d6afb2aeabba240898512ef274ffe570ef59c9ba5401cd741fd
  Gas price: 0.0 gwei   Gas limit: 6721975   Nonce: 1
  UpkeepManager.addMember confirmed (reverted)   Block: 16928939   Gas used: 143785 (2.14%)

  File "brownie/_cli/run.py", line 50, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/issue/1198/fee_module_integration.py", line 49, in add_member
    upkeep_manager.addMember(
  File "brownie/network/contract.py", line 1622, in __call__
    return self.transact(*args)
  File "brownie/network/contract.py", line 1495, in transact
    return tx["from"].transfer(
  File "brownie/network/account.py", line 680, in transfer
    receipt._raise_if_reverted(exc)
  File "brownie/network/transaction.py", line 421, in _raise_if_reverted
    raise exc._with_attr(
VirtualMachineError: revert
Trace step 6442, program counter 649:
  File "EACAggregatorProxy-flattened.sol", lines 776-791, in EACAggregatorProxy.latestRoundData:    
      */
      function proposedGetRoundData(uint80 _roundId)
        public
        view
        checkAccess()
        hasProposal()
        override
        returns (
          uint80 roundId,
          int256 answer,
          uint256 startedAt,
          uint256 updatedAt,
          uint80 answeredInRound
        )
      {
        return super.proposedGetRoundData(_roundId);
      }
    
Terminating local RPC client...
```

**Comment by petrovska-petro** [comment]
> getting this on add_member??

That was due to the mngr not having enough funds, i topped it up today. now should be g2g. 

Tested it when *fast wei oracle* was reporting 36.3 gwei

https://dashboard.tenderly.co/testing-ape/project/simulator/c12b9f99-0327-4167-9d72-dfe4110df159/logs

**Review by gosuto-inzasheru — APPROVED** [review]
runs successfully but only at low gwei. otherwise upkeep manager needs more eth balance to support the `add_member`

File: great_ape_safe/ape_api/badger.py [hunk 1]
```diff
@@ -424,7 +424,8 @@ def set_gas_station_watchlist(self):
             if "executor" in label:
                 min_bal = 1e18
             if label == "upkeep_manager":
-                min_bal = 1e18
+                min_bal = 3e18
+                min_top_up = 2e18
```
Comment (gosuto-inzasheru) [inline great_ape_safe/ape_api/badger.py:428]:
```suggestion
                min_bal = 5e18
                min_top_up = 4e18
```

Comment (gosuto-inzasheru) [reply great_ape_safe/ape_api/badger.py:428]: with gwei at 42 and topping up upkeep manager manually with 2 extra eth the script still reverts (ie mngr needs >2.6eth)

given that in the future average gwei might go up more, and that we might want to deploy other upkeeps at >2m gas, lets give upkeep manager a nice buffer.

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]
lgtm

**Pull request merged**